src/Volume.ts

```diff
@@ -262,15 +300,23 @@ export default class Volume {
     // While we're here, pick a power of 10 that divides into our max dimension a reasonable number of times
     // and save it to be the length of tick marks in 3d.
     this.tickMarkPhysicalLength = 10 ** Math.floor(Math.log10(this.physicalScale / 2));
-
-    // sx, sy, sz should be same as normalizedPhysicalSize
-    this.scale = new Vector3(sx, sy, sz);
   }
 
   setUnitSymbol(symbol: string): void {
     this.physicalUnitSymbol = symbol;
   }
 
+  /** Computes the center of the volume subset */
+  getContentCenter(): Vector3 {
+    // center point: (contentSize / 2 + contentOffset - 0.5) * normalizedPhysicalSize;
```

Comment:
I like having the math notation in a comment above.  So much more readable than the threejs functions.